tests/App/config.yml

```diff
@@ -6,8 +6,10 @@ framework:
         enabled: true
 
 twig:
+    exception_controller: null
```

Comment:
```suggestion
    exception_controller: ~
```

Reply:
I used to use `~`, but I've seen sometimes it is not clear its meaning, so I think using `null` is more straightforward.

Reply:
@phansys I'll let you merge if you agree, I don't have a strong opinion on this.

Reply:
@franmomu do you mean the specification says it should be interpreted as null in some contexts, or just that not everybody know that this means `null`?

Reply:
> @franmomu do you mean the specification says it should be interpreted as null in some contexts, or just that not everybody know that this means `null`?

The second one, apparently looks like Symfony were changing `~` to `null` in the recipes https://github.com/symfony/recipes/pull/635

Reply:
Let's merge then :+1: